packages/router/__tests__/router-test.ts

```diff
@@ -1144,29 +1144,55 @@ describe("a router", () => {
     it("reloads all routes if X-Remix-Revalidate was set in a loader redirect header", async () => {
       let t = initializeTmTest();
 
-      let A = await t.navigate("/bar");
+      let A = await t.navigate("/foo");
```

Comment:
Made this test go `foo->bar` instead of `bar->baz` to match up with the below test